cmd/kubevirt-operator/Dockerfile

```diff
@@ -0,0 +1,32 @@
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2018 Red Hat, Inc.
+#
+
+
+FROM centos:7
```

Comment:
Can you use `fedora:28`?

Reply:
just curios, if fedora, why not 29?

Reply:
Sure.

Reply:
> just curios, if fedora, why not 29?

Because all other containers use it right now. We need to update all together.